tests/test_debug/test_snapshot.py

```diff
@@ -216,33 +212,31 @@ def test_snapshot_to_json_tuples_primitive():
 
     assert json_data_frames == [
         {
-            "id": None,
-            "isClass": True,
+            "type": ".frame",
             "name": "func1",
-            "stack_frame": True,
+            "id": None,
```

Comment:
this change is showing up because you reordered this entry, but you should move it back to its original spot to ensure the diff is as small as possible

(same as in the other test cases below)

Reply:
Done!